src/frontend/src/pages/myfeeds.js

```diff
@@ -131,6 +135,9 @@ export default function MyFeeds() {
       setSubmitStatus({ message: error.message, isError: true });
       console.log('Error adding feeds', error);
     }
+    // Adding these to change state and cause snackbar to re-render
+    setCount(count + 1);
+    setAlert(false);
```

Comment:
This part still confuses me the most, it is not enough I setAlert(false), I also have to change another state on top of this.